site/src/doc/tutorials/background.md

```diff
@@ -36,12 +36,12 @@ For convenience, these are also available as named exports, e.g.
 
 ```js
 import {Map, View} from 'ol';
-import {Tile, Vector} from 'ol/layer';
+import {Tile, Vector} from 'ol/layer.js';
 ```
 
 In addition to these re-exported classes, modules with lowercase names also provide constants or functions as named exports:
 
 ```js
 import {getUid} from 'ol';
```

Comment:
Should these two imports of ol also be changed? It was changed in quickstart.md.

Reply:
I’m not seeing what you are referring to.  But pleased follow up with a ticket or pull request if there is more to change here.